app/src/main/java/com/beemdevelopment/aegis/importers/SqlCipherImporterHelper.java

```diff
@@ -0,0 +1,115 @@
+package com.beemdevelopment.aegis.importers;
```

Comment:
Can you remove the unused imports?

Reply:
Will tackle it when I refactor the classes anyway.